### PR TITLE
Change behaviour of split variable

### LINKED ISSUE #25: Change default behavior for `split=None` in training of subject_level
For patch and slice, if `split = None` all the splits of the k-fold CV are done. It could be implemented for subject level and default value for `split` could be set to `None`.
Suggested in PR #20.

### PR DESCRIPTION
**Pull request by 14thibea**
This PR closes https://github.com/aramis-lab/AD-DL/issues/25.

`split` is now a list of folds that are executed (default behaviour, all splits are trained).
As one run may correspond to several folds, the json file was moved at the root.

### CODE REVIEW

File: clinicadl/clinicadl/image_level/utils.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 # CNN train / test  #
 #####################
 
-def train(model, train_loader, valid_loader, criterion, optimizer, resume, options):
+def train(model, train_loader, valid_loader, criterion, optimizer, resume, split, options):
```
Comment (mdiazmel) [inline clinicadl/clinicadl/image_level/utils.py]: Why `split` is an additional parameter of this function? This information is not embedded in the `options` data structure?
I thing that we should keep the "training" functions with few arguments as possible and let the user of the function to manage the naming of the folders, files.

**Review by mdiazmel — COMMENTED** [review]
Hi @14thibea,
The `split` parameter is now mandatory for train function on `image_level` (see below). Is this necessary? I wonder if the naming of  the folders/files can be done outside of the training functions in order to keep these functions as simple as possible.

File: clinicadl/clinicadl/image_level/utils.py (hunk 1 above)
Comment (14thibea) [reply clinicadl/clinicadl/image_level/utils.py]: Actually options.split contains the list of splits that must be done in the training, and train execute the computations for one particular split (so the arguments are different).
I can detach the naming of the folders / writers etc from the function so it will become independent from the split as you suggested !

Comment (14thibea) [reply clinicadl/clinicadl/image_level/utils.py]: Done in https://github.com/aramis-lab/AD-DL/pull/41/commits/1b5848855f1b1bbe65fcf7a9746a0fb7121bf59a
But now there are more arguments in train...

Comment (mdiazmel) [reply clinicadl/clinicadl/image_level/utils.py]: Yes, I see.... but I thing that this is better for visibility/control of files/folder created during the training.

**Comment by mdiazmel** [comment]
It seems that this PR is ready to merge? Can you confirm it, @14thibea ?

**Comment by 14thibea** [comment]
It's ok, the tests all passed !

**Comment by mdiazmel** [comment]
I'll merge it!
